### ISSUE TITLE
Music player embed.

### ISSUE THREAD

**Issue by DolphinDTM** [zardoru/becgi#19]
Either allowing DDL links to mp3/ogg/wav or SoundCloud embeds, preferably both.
I know the concern regarding SoundCloud embeds was that you potentially needed to resolve the song ID via Javascript which could lead to code injection.

**Comment by zardoru** [comment]
WRT soundcloud embeds: I think we can get around it by validating URLs.
DDL links to mp3/ogg/wav should be doable by an audio tag.
For METTATUNES I think we can settle for markdown? In any case, this has priority above the other embeds.

**Comment by DolphinDTM** [comment]
I didn't know it was possible to embed either in Markdown or am I misunderstanding?

**Comment by zardoru** [comment]
Not really, but you can build links at least. [like this](http://example.com)

**Comment by DolphinDTM** [comment]
Fair. Embeds are not a huge priority anyway.
